File: CP/CPP/Week 3/A_Equation.cpp
// https://codeforces.com/problemset/problem/1269/A

#include<iostream>
using namespace std;
int isprime(int n)
{
    if( n <= 2) 
        return true;
    for(int i = 2; i * i <= n; i++)
        if(n%i == 0)
            return false;
    return true;
}
int main()
{
    int n; 
    cin>>n;
    for(int x = 4; x <= 1000; x++)
    {
        int a = n + x;
        int b = x;
        if(isprime(a) || isprime(b)) continue;
        if(a - b == n)
        {
            cout<<a<<" "<<b<<endl;
            return 0 ;
        }
    }
    return 0;
}

/*

input
1
output
9 8

input
512
output
4608 4096

*/

File: CP/Java/Week 3/A_Equation.java
// https://codeforces.com/problemset/problem/1269/A

import java.util.Scanner;
public class A_Equation {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        long n = scanner.nextLong();

        if (n == 1) {
            System.out.println(9 + " " + 8);
        } else {
            System.out.println(n * 4 + " " + n * 3);
        }
    }
}
/*
Input
1
Output
9 8

 */
